### DISCUSSION TITLE
List of all LVars for A32NX

### DISCUSSION THREAD

**Discussion by roarkr** [flybywiresim/aircraft#6001] [Help] [answered]
Where can I find a text file of all used A32NX Lvars, the same list I can view in the MSFS debug window's LocalVariables?

**Comment by Benjozork** [comment, marked as answer]
https://github.com/flybywiresim/a32nx/blob/master/docs/a320-simvars.md

**Reply by roarkr** [reply]
Thanks for your reply. 
I know that LVar list, but there are many more LVars for the A32NX that are not listed in that document.
The LVars I am missing are all the XMLVARs.

**Reply by tracernz** [reply]
> Thanks for your reply. I know that LVar list, but there are many more LVars for the A32NX that are not listed in that document. The LVars I am missing are all the XMLVARs.

Those are Asobo A320 ones that have not been replaced yet. Unfortunately we won't document those.
